src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
This prevents `updatesPerSecond` from being larger than `fps`. Because the `stepSize` is capped to `fps`, it makes the new while loop in `me.game.update` useless; `updateDelta` can never be higher than ~16.6667, which is roughly what `me.timer.getDelta()` returns during RAF.

In other words, this use of `Math.min` limits the while loop to acting as an if-statement; it will only execute the body zero or one time.